app/common/update.py

```diff
@@ -261,62 +261,87 @@ def merge_local_db():
         
 
 def download_dat_files():
+    """
+    Verifies the user's DQX install location and prompts
+    them to locate it if not found. Uses this location to
+    download the latest data files from the dqxclarity repo.
+    """
     config = load_user_config()
-    valid_directory = False
+    DAT0_FILE = "data00000000.win32.dat0"

```

Comment:
We should make these two lower case. Capital variable names indicate global variable names, which are defined outside of functions.